src/gui/src/clockWidget.cpp

```diff
@@ -847,9 +847,13 @@ bool ClockTreeView::changeSelection(const SelectionSet& selections)
   if (!nodes.empty()) {
     // remove old selection
     clearSelection();
+
+    // prevent ClockTreeView to draw until unlock
+    lockRender();
     for (auto node : nodes) {
       node->setSelected(true);
     }
+    unlockRender();
```

Comment:
when this is done, you need to call `selectionChanges()` otherwise all the selection emits from `canConvertAndEmit` will not be processed.